src/main/java/org/folio/dataexp/service/logs/ErrorLogService.java

```diff
@@ -238,6 +238,8 @@ private ErrorLog getGeneralErrorLog(String errorMessageCode, UUID jobExecutionId
 
   private String getInventoryRecordLink() {
      var linkFromTheStorage = configurationService.getValue(INVENTORY_RECORD_LINK_KEY);
-     return (linkFromTheStorage.endsWith("/") ? linkFromTheStorage : linkFromTheStorage.concat("/"));
+     var uri = (linkFromTheStorage.endsWith("/") ? linkFromTheStorage : linkFromTheStorage.concat("/"));
+
+    return String.join(uri, "inventory/view/");
```

Comment:
I think the idea to have correct link in INVENTORY_RECORD_LINK_KEY and do not hardcode, why not to save it in config table during table enable? together with it you can rename INVENTORY_RECORD_LINK_KEY from folio_host to inventory_record_link

Reply:
done